app/views/events/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2 class="text-primary text-center">Events List</h2>
+  <table class="table table-striped border">
+  <thead>
+    <tr>
+      <th scope="col">Name</th>
+      <th scope="col">Description</th>
+      <th scope="col">Date</th>
+      <th scope="col">Creator</th>
+      <th scope="col"></th>
+    </th>
+  </thead>
+  <tbody>
+    <% @events.each do |e| %>
```

Comment:
- [ ] There should be no logic at all( if else, each, etc) in the views.
View logic should be inside the pertinent helper.